nixos/tests/gnome3-xorg.nix

```diff
@@ -1,7 +1,7 @@
 import ./make-test.nix ({ pkgs, ...} : {
   name = "gnome3-xorg";
   meta = with pkgs.stdenv.lib.maintainers; {
-    maintainers = [ domenkozar eelco lethalman ];
+    maintainers = pkgs.gnome3.maintainers;
```

Comment:
Should we remove Eelco and Domen here?

Reply:
Have they contributed to the test in the past year? Would they be responsive to pings to fix and review it?